src/commands/dev/dev.mjs

```diff
@@ -161,7 +161,10 @@ const dev = async (options, command) => {
     },
   })
 
+  const geoConfig = { geolocationMode: options.geo, geoCountry: options.country, offline: options.offline, state }
```

Comment:
Why are `offline` and `state` part of an object called `geoConfig`?

Reply:
Because these fields are all required by the `getGeoLocation` function:

https://github.com/netlify/cli/blob/5ccdc7115427356f91643120949cce58c6c8513a/src/lib/geo-location.mjs#L47

Reply:
Sure, but these are generic properties that are present in most commands, not specific to geolocation. I think we should be grouping parameters based on what they mean for the function being called, not on how they will be used by a child function many levels deep.

If `startFunctionsServer` (or any other methods downstream) start using `offline` or `state` for other purposes than geo-location, would we move them out of this object?

Not a big deal, just think our future selves might find this confusing.

Reply:
Yeah, fair point. Refactored in https://github.com/netlify/cli/pull/5804/commits/adbb13ec68945fb524cb1573213307873ac78527.